src/docs/get-started/install/_get-sdk-win.md

```diff
@@ -13,6 +13,14 @@
     do not install Flutter in a directory like
     `C:\Program Files\` that requires elevated privileges).
 
+Steps 1 and 2 can be replaced. If you don't want to install a fixed version of installation bundle.
+Flutter SDK is free and open source, so we can get the source code from [GitHub](https://github.com/flutter/flutter),
```

Comment:
Please make these changes to both files.

a fixed version of installation bundle => a fixed version of the installation bundle
Flutter SDK is free => The Flutter SDK is free
so we can => so you can
get the source code from [GitHub] => get the source code from the [Flutter repo]() on GitHub.

Reply:
Thanks for your reminder Sfshaza.